api/src/templates/projects.html

```diff
@@ -82,5 +38,56 @@ <h1>Projects</h1>
     {% endfor %}
    </ul>
   </div>
+
+  <form class="create-new-project" action="/api/forms/projects" method="post">
```

Comment:
I was getting 404 not found trying to create a new project. Changing the action attribute to ```/forms/projects``` fixed it. I think this was from trying to move forms to its own container.

Reply:
Right. Like I was saying above, I wasn't sure what the intended endpoint url should be. Do you want it to be /forms/projects etc. for the forms or prepended with /api/ as in /api/forms/projects like the other endpoints are? I think if you didn't want the api prepended to the forms it's a matter of just changing some URLs in the forms which I can go ahead and do. Should I go ahead and do that for the forms?

Reply:
I'm leaning towards sticking with /forms/projects for now. It seems to be the path of least resistance to a working solution. Eventually, if forms run in their own container, and calls the api through nginx, we can change it to /api/forms/projects.

Reply:
Ok, that makes sense. I'll go ahead with the /forms/* endpoints then.

Reply:
Actually, I'm getting the opposite behavior with the forms - the above action attribute works with /api/forms/projects. It's just that the way /api/ is prepended in Flask is inconsistent - sometimes it is automatic and sometimes it isn't. So, if you're ok with just getting something up and going I can go ahead and get everything linked up. But I think it will be with /api/forms/* which would match the other endpoints better I think. There are a couple places in the Flask routes that I would need to update which maybe was why it wasn't working for you (adding a project wasn't redirecting correctly, e.g.). I'll let you know when I get something working.

Reply:
Ok, the forms should be working correctly now, including the links/URLs. Let me know if you find anything not working. I noticed there is no delete project functionality. Should I add that? Next action for me I think should be validation on these forms - front end and back end....

Reply:
I added client-side validation with html constraints. Next I will add some server-side validation with FlaskWTF. Do you think we need to add any other specific validation or JS client-side validation?